draft-ietf-quic-load-balancers.md

```diff
@@ -497,21 +497,20 @@ The expand(length, pass, input_bytes) function concatenates three arguments and
 outputs 16 zero-padded octets.
 
 The first argument 'length' is an 8-bit integer that reports the sum of the
-configured nonce length and server id length in octets, and forms the most
-significant octet of the output. The 'length' argument MUST NOT exceed 28.
+configured nonce length and server id length in octets, and forms the fifteenth
+octet of the output. The 'length' argument MUST NOT exceed 28.
 
 The second argument is an 8-bit integer that is the 'pass' of the algorithm, and
-forms the second-most significant octet of the output.
+forms the sixteenth (least significant) octet of the output.
 
 The third argument is a variable-length stream of octets, which is copied into
-the third-most significant octet of the output and beyond. The length of this
-octet stream is half the 'length', rounded up. All remaining octets of the
-output are zero.
+the most significant octets of the output. The length of this octet stream is
+half the 'length', rounded up. All remaining octets of the output are zero.
 
```

Comment:
This appears to be half an edit. The syntax as you express it appears to be:
~~~
ExpandResult {
      Bytes(...),
      ZeroPad(...),
      length(8),
      pass(8)
}
~~~

In which:

* 'Bytes' are the N most significant octets of the output. The number N is half the 'length', rounded up, thus a number between 1 and 14.
* 'Zeropad' is a set of 14-N zeroes,
* 'length' is an 8-bit integer that reports the sum of the configured nonce length and server id length in octets, and forms the fifteenth octet of the output. The 'length' argument MUST NOT exceed 28.
* 'pass' is an 8-bit integer that reports the 'pass' argument of the algorithm, and
forms the sixteenth (least significant) octet of the output.

Reply:
OK, I took this working, minus some corrections. Length has to be between 5 and 19, meaning N is between 3 and 10.